.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,43 @@
+name: Docker Build
+
+on:
```

Comment:
You would want this on any change, i.e. if you update a component or page it should still rebuild right?

Reply:
So this rebuild action is not just for CSV changes?

Reply:
Updated.

Reply:
Might want to restrict this to the main branch, otherwise you will be updating the current image with your changes

Reply:
Fixed